core/templates/dev/head/pages/exploration-player-page/learner-experience/learner-answer-info-card.directive.html

```diff
@@ -0,0 +1,43 @@
+<div class="learner-answer-info-card">
+  <div class="enter-answer-details">
+    <textarea class="form-control" ng-model="$ctrl.answerDetails" rows="4" placeholder="Enter here"></textarea>
+  </div>
+  <div class="submit-button">
+    <md-button class="oppia-learner-confirm-button" ng-disabled="!($ctrl.answerDetails)" ng-click="$ctrl.submitLearnerAnswerInfo()">SUBMIT</md-button>
+  </div>
+</div>
+<style>
+  .learner-answer-info-card {
```

Comment:
Prefix all Oppia classes with "oppia-", here and elsewhere.

Reply:
Done